Slam/Queue/QueueView.swift

```diff
@@ -24,4 +30,16 @@ public class QueueView: NSObject, UITableViewDataSource {
         return cell
     }
 
+    public func tableView(tableView: UITableView, commitEditingStyle editingStyle: UITableViewCellEditingStyle, forRowAtIndexPath indexPath: NSIndexPath) {
+        if editingStyle == UITableViewCellEditingStyle.Delete {
+            var match = matches[indexPath.row]
```

Comment:
This could be a `let` instead of a `var`. In general, we should use `let` when we can.

Also, there is potential duplication between this line and the first line in `tableView cellForRowAtIndexPath`. (Line 27)